### ISSUE TITLE
Insert Line Break Method?

### ISSUE THREAD

**Issue by msf567** [lukemalcolm/ofxTextSuite#8]
Hi!

I'm looking for a way to just add a line break to a block of text, however there is no such method in the header. I have tried writing my own by looking at the wrap text methods, but to no avail.

How would I go about forcing a newline in a textBlock object?

**Comment by Vamoss** [comment]
See this version, it also have a html tag format method.
https://github.com/Vamoss/ofxTextSuite

**Comment by msf567** [comment]
Sure, but how would I go about adding a line break with each occurrence of '\n' in the textblock's text member?

**Comment by Vamoss** [comment]
You are right, I did not pushed this feature. I will look for it tomorrow. Best!

**Comment by Vamoss** [comment]
Just updated the break line functionality:
https://github.com/Vamoss/ofxTextSuite/commit/d3f65c983672fe410234fa591abf1025aba4e4d8

The htmlExample show how to use. Best!
